finality-aleph/src/sync/ticker.rs

```diff
@@ -0,0 +1,143 @@
+use tokio::time::{sleep, Duration, Instant};
+
+/// This struct is used for determining when we should broadcast justification so that it does not happen too often.
```

Comment:
This will not only be about broadcasting justifications, so it would be nice if you could make all the comments more generic – just that we don't want to this to trigger more often than once every `min_timeout`, no less often than once every `max_timeout`, and that there is a special request for an asap tick. Maybe you could even avoid mentioning that this is for broadcasts at all, it technically is just a rate-limited on-demand ticker.

Anyway, this is just an extensive nit, apply however much or little of it as you want (although I would at least generalize away from justifications, otherwise it will have to be changed later).

Reply:
Changed after renaming to `Ticker`